src/main/java/com/kit/gui/controller/LoginController.java

```diff
@@ -133,6 +133,7 @@ private boolean loadAccount(String uuid, boolean rememberMe, String email) throw
                         apiKeyProperty.remove();
                     }
                 }
+                Session.get().onAuthenticated();
```

Comment:
Wonder if it's better to use our event bus here, but then maybe it shouldn't be used for these types of events